modules/migration-mtc-support.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="migration-mtc-support_{context}"]
+= {mtc-short} 1.8 support matrix
+
+.Support matrix based on {oadp-first} and {OCP} dependency
+[cols=",,,",options="header",]
+|===
+|{mtc-short} version
+|{oadp-short} version
+|{OCP-short} version
+|Supported
+
+|{mtc-short} 1.8.0
+|{oadp-short} 1.3.z and later
+|{OCP-short} 4.15 and earlier
+|Yes
+
+|{mtc-short} 1.8.0
+|{oadp-short} 1.4.z and later
+|{OCP-short} 4.16 and earlier
+|No
+
+|{mtc-short} 1.8.1
+|{oadp-short} 1.3.z and later
+|{OCP-short} 4.15 and earlier
+|Yes
+
+|{mtc-short} 1.8.1
+|{oadp-short} 1.4.z and later
+|{OCP-short} 4.16 and earlier
+|No
+
+|{mtc-short} 1.8.2
+|{oadp-short} 1.3.z and later
+|{OCP-short} 4.15 and earlier
+|Yes
+
+|{mtc-short} 1.8.2
+|{oadp-short} 1.4.z and later
+|{OCP-short} 4.16 and earlier
+|No
+
+|{mtc-short} 1.8.3
+|{oadp-short} 1.3.z and later
+|{OCP-short} 4.15 and earlier
+|Yes
+
+|{mtc-short} 1.8.3
+|{oadp-short} 1.4.z and later
+|{OCP-short} 4.16 and earlier
+|No
+
+|{mtc-short} 1.8.4
+|{oadp-short} 1.4.z and later
+|{OCP-short} 4.16 and later
+|Yes
+|===
+
```

Comment:
```suggestion
.Support matrix based on {oadp-first} and {OCP} dependency
[cols=",,,",options="header",]
|===
|{mtc-short} version
|{oadp-short} version
|{OCP-short} version
|Supported

|{mtc-short} 1.8.0
|{oadp-short} 1.3.z and later
|{OCP-short} 4.15 and earlier
|Yes

|{mtc-short} 1.8.0
|{oadp-short} 1.4.z and later
|{OCP-short} 4.16 and earlier
|No

|{mtc-short} 1.8.1
|{oadp-short} 1.3.z and later
|{OCP-short} 4.15 and earlier
|Yes

|{mtc-short} 1.8.1
|{oadp-short} 1.4.z and later
|{OCP-short} 4.16 and earlier
|No

|{mtc-short} 1.8.2
|{oadp-short} 1.3.z and later
|{OCP-short} 4.15 and earlier
|Yes

|{mtc-short} 1.8.2
|{oadp-short} 1.4.z and later
|{OCP-short} 4.16 and earlier
|No

|{mtc-short} 1.8.3
|{oadp-short} 1.3.z and later
|{OCP-short} 4.15 and earlier
|Yes

|{mtc-short} 1.8.3
|{oadp-short} 1.4.z and later
|{OCP-short} 4.16 and earlier
|No

|{mtc-short} 1.8.4
|{oadp-short} 1.3.z and later
|{OCP-short} 4.15 and earlier
|Yes


|{mtc-short} 1.8.4
|{oadp-short} 1.4.z and later
|{OCP-short} 4.16 and later
|Yes
|===

```


>could we include something like this in the documentation to highlight the dependency issue WDYT Thanks

@anarnold97 , not really sure how best to propose the table/matrix.  If this table, per each MTC release is the path forward, I'll `+1` it.

The short version is:
- MTC <=1.8.3 using OADP 1.3.z (<1.4.0) is supported on all [supported] OCP versions <=4.15
- MTC >=1.8.4 using OADP 1.3.z (<1.4.0) -- as of right now -- is supported on all [supported] OCP versions <=4.15
- MTC >=1.8.4 using OADP 1.4.z (>=1.4.0 <1.5.0) -- as of right now -- is supported on all [supported] OCP versions >=4.13

Reply:
Yes, I find the matrix confusing to be honest. It should be limited to highlight the important points.

Reply:
Cool enough. Updated